api/utilities/retrieval_system.py

```
@@ -0,0 +1,85 @@
+    - document_embeddings: Embeddings for raw_chunks (used in FAISS)
+    - chunk_labels: List of labels for each chunk
+    - top_n: Number of top chunks to return
+    - alpha, beta, gamma: Weights for FAISS, BM25, and label similarity scores
+    - similarity_threshold: Minimum semantic similarity between query label and chunk label
+
+    Returns:
+    - List of top_n most relevant raw chunks
+    """
+    import numpy as np
+    from nltk.tokenize import word_tokenize
+    from sentence_transformers import SentenceTransformer
```

Ruff (0.11.9)

`sentence_transformers.SentenceTransformer` imported but unused

Remove unused import: `sentence_transformers.SentenceTransformer`

(F401)